indexer_util/indexer_util/indexer_util.py

```diff
@@ -101,3 +102,22 @@ def maybe_create_elasticsearch_index(elasticsearch_url, index_name):
             'Creating %s index at %s.' % (index_name, elasticsearch_url))
         es.indices.create(index=index_name, body={})
     return es
+
+
+def bulk_index(es, index, docs_by_id):
+    # Use generator so we can index arbitrarily large iterators (like tables),
+    # without having to load into memory.
+    def es_actions(docs_by_id):
+        for key, doc in docs_by_id:
```

Comment:
nit key -> id

Reply:
Done (changed to `_id`, `id` is reserved in python)